_includes/pbc/all/install-features/202304.0/install-the-agent-assist-feature.md

```diff
@@ -199,6 +199,20 @@ Make sure that when the login form for the agent is submitted, the URL it uses c
 
 {% endinfo_block %}
 
+Make sure to use environment variables in `config-default.php`:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+// other code
+
+// >>> Security Blocker Storefront Agent
+$config[SecurityBlockerStorefrontAgentConstants::AGENT_BLOCK_FOR_SECONDS] = 360;
```

Comment:
config constants are present at respective table for `SecurityBlockerBackofficeConstants` case, but these tables are missing for other cases